src/App.vue

```diff
@@ -26,22 +26,19 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from "vue";
-import Header from "@/components/Header.vue";
+import { defineComponent, defineAsyncComponent } from "vue";
 // import Search from "@/components/Search.vue";
-import RealTimeMonitor from "@/components/RTM/RealTimeMonitor.vue";
-import Navigation from "@/components/Navigation.vue";
-import FooterShard from "@/components/Footer.vue";
 import './assets/styles/tailwind.css';
+import Header from "@/components/Header.vue";
 
 export default defineComponent({
   name: "App",
   components: {
     Header,
     // Search,
-    RealTimeMonitor,
-    Navigation,
-    FooterShard,
+    RealTimeMonitor: defineAsyncComponent(() => import(/* webpackChunkName: "RealTimeMonitor" */ "@/components/RTM/RealTimeMonitor.vue")),
```

Comment:
I would keep these components registered locally instead of splitting them as there is nothing to gain only lose. If you have components that are used in multiple places consider registering them as a global component in the main Vue instance.

Also the chunk name doesn't add anything in this case. I would reserve that for components that you split of and want to load together.

Reply:
Fair enough, corrected in https://github.com/ps2alerts/website/pull/61/commits/211e5c52d2ddda189065413fc580ca690fc8c14a